### PR TITLE
Feature/recieve notifications

### PR DESCRIPTION
**Pull request by dariokozicki**
## Links:
image

## What & Why:
When creating a target, if a match is made then a dialog will popup to take the user to chat with their match.

image

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/components/common/NewMatch/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    dispatch(setConversationSelected(newMatch.match_conversation?.id));
+    dispatch(setHomeTab(tabsEnum.chat));
+    dispatch(setNewMatch(null));
+  };
+
+  const onSkip = () => dispatch(setNewMatch(null));
+
+  return (
+    <>
+      <div className="new-match-dialog">
```
Comment (tobicorradi) [inline src/components/common/NewMatch/index.js:28]: WDYT if you replace all those divs tags with some semantic HTML tags? Like section or article?

**Review by tobicorradi — APPROVED** [review]
LGTM! Good job 🚀

File: src/components/common/Tabs/ChatTab/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      setMatch(conversations.matches.find(m => m.match_id === conversationId) || {});
+    }
+  }, [conversationId, conversations]);
+
+  const onBack = () => {
+    dispatch(setHomeTab(tabsEnum.profile));
+  };
+
+  const handleReceivedChat = data => {
+    console.log('recibi un mensaje', data);
```
Comment (tobicorradi) [inline src/components/common/Tabs/ChatTab/index.js:35]: And remember to delete those logs after

Comment (dariokozicki) [reply src/components/common/Tabs/ChatTab/index.js:35]: Yeah sadly this is from an incomplete task that I had trouble with and I'll need to finish in the future (actioncable)

File: src/components/common/NewMatch/index.js (hunk 1 above)
Comment (dariokozicki) [reply src/components/common/NewMatch/index.js:28]: Will look into it! I definitely overuse divs as a one trick pony

File: src/components/common/Tabs/ChatTab/index.js (hunk 2 above)
Comment (tobicorradi) [reply src/components/common/Tabs/ChatTab/index.js:35]: Sure, don't worry!

**Pull request merged**